src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/Attachment.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.entity;
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import org.springframework.data.mongodb.core.mapping.Field;
+import uk.gov.companieshouse.service.links.Links;
+
+import java.io.Serializable;
+
+public class Attachment implements Serializable {
+
+    @Field("id")
+    private String id;
+    @Field("links")
+    @JsonUnwrapped
```

Comment:
We want the links to be in their own Json links object so i don't think we'd want to unwrap them
https://fasterxml.github.io/jackson-annotations/javadoc/2.8/com/fasterxml/jackson/annotation/JsonUnwrapped.html

Reply:
Some more info: CH API guidelines and examples https://companieshouse.atlassian.net/wiki/spaces/DEV/pages/50169325/API+Design+Guidelines

Reply:
Have removed this annotation - we are unwrapping in extensions is this something we will want to change at some point?

Reply:
i don't know.  it should look like this though
links: {
    self: "/customer/12345",
    customer_orders: "/customer/12345/orders",
    customer_account: "/customer/12345/account"
}